cdap-client/src/main/java/co/cask/cdap/client/config/ConnectionConfig.java

```diff
@@ -66,10 +67,11 @@ public ConnectionConfig(Id.Namespace namespace, String hostname, int port, boole
     this.hostname = hostname;
     this.port = port;
     this.sslEnabled = sslEnabled;
+    this.username = System.getProperty("user.name");
```

Comment:
And in the case of insecure CDAP, don't show username at all, right?
Since the CLI user is not logged into a CDAP user.